### PR TITLE
Krushmedia: update user sync urls

### PR DESCRIPTION
**Pull request by Krushmedia**

### CODE REVIEW

**Comment by Krushmedia** [comment]
@bsardo friendly ping. Review this pull request please

File: static/bidder-info/krushmedia.yaml [hunk 1]
```diff
@@ -14,5 +14,8 @@ capabilities:
       - native
 userSync:
   redirect:
-    url: "https://cs.krushmedia.com/4e4abdd5ecc661643458a730b1aa927d.gif?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&redir={{.RedirectURL}}"
+    url: "https://cs.krushmedia.com/pbserver?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&ccpa={{.USPrivacy}}&gpp={{.GPP}}&gpp_sid={{.GPPSID}}&redir={{.RedirectURL}}"
+    userMacro: "[UID]"
```
Comment (bsardo) [inline static/bidder-info/krushmedia.yaml:18]: Confirmed redirect is working
Screenshot 2024-11-04 at 5 01 02 PM

File: static/bidder-info/krushmedia.yaml [hunk 2]
```diff
@@ -14,5 +14,8 @@ capabilities:
...
+  iframe:
+    url: "https://cs.krushmedia.com/pbserverIframe?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&ccpa={{.USPrivacy}}&gpp={{.GPP}}&gpp_sid={{.GPPSID}}&pbserverUrl={{.RedirectURL}}"
```
Comment (bsardo) [inline static/bidder-info/krushmedia.yaml:20]: Confirmed iframe is working
Screenshot 2024-11-04 at 5 05 33 PM

**Review by bsardo — APPROVED** [review]

**Review by przemkaczmarek — APPROVED** [review]

**Comment by Krushmedia** [comment]
@bsardo @przemkaczmarek thanks for the approval. Will these changes make it into the next release?

**Pull request merged**